internal/sbi/producer/data_repository.go

```diff
@@ -305,6 +305,10 @@ func HandleQueryAuthSubsData(request *httpwrapper.Request) *httpwrapper.Response
 func QueryAuthSubsDataProcedure(collName string, ueId string) (map[string]interface{}, *models.ProblemDetails) {
 	filter := bson.M{"ueId": ueId}
 	data, pd := getDataFromDB(collName, filter)
+	if pd != nil && pd.Status == 404 {
```

Comment:
why not check the Status of ProblemDetails in udm directly? It returns the same thing.

Reply:
It is better to replace `404` with `http.StatusNotFound`.

Reply:
And can combine these 2 if:
```
	if pd != nil {
		if pd.Status == http.StatusNotFound {
			logger.DataRepoLog.Warnf("QueryAuthSubsDataProcedure err: %s", pd.Title)
		} else {
			logger.DataRepoLog.Errorf("QueryAuthSubsDataProcedure err: %s", pd.Detail)
		}
		return nil, pd
	}
```

Reply:
I have made this modification because I have decided that in the case of per-user errors such as this one, a log level of Warning is more appropriate than Error.

And I have fixed and PR as you suggested.